vita3k/renderer/src/gl/compile_program.cpp

```diff
@@ -193,15 +193,15 @@ void pre_compile_program(GLState &renderer, const char *base_path, const char *t
         // Compile Fragment Shader
         const auto frag_hash_hex = convert_hash_to_hex(hash.frag);
         const SharedGLObject frag_shader = compile_shader(base_path, title_id, self_name, renderer.shader_version,
-            frag_hash_hex.c_str(), "frag", GL_FRAGMENT_SHADER, renderer.fragment_shader_cache, hash.frag);
+            frag_hash_hex, "frag", GL_FRAGMENT_SHADER, renderer.fragment_shader_cache, hash.frag);
```

Comment:
is wrong here, this function requestconst char* you need using c_str()

Reply:
Wrong, it requires a const std::string&, not a const char*, GermanAizek is right

Reply:
ah if you said, i have mistake avec othe rin this